src/services/types.ts

```diff
@@ -19,34 +19,34 @@ namespace ts {
         getFirstToken(sourceFile?: SourceFile): Node;
         getLastToken(sourceFile?: SourceFile): Node;
         // See ts.forEachChild for documentation.
-        forEachChild<T>(cbNode: (node: Node) => T, cbNodeArray?: (nodes: Node[]) => T): T;
+        forEachChild<T>(cbNode: (node: Node) => T, cbNodeArray?: (nodes: Node[]) => T): T | undefined;

```

Comment:
Just for documentation: if a callback always returns a truthy value then `forEachChild` will terminate immediately and return the first result. Since it's always allowed to return undefined, it doesn't hurt to have it there explicitly.